kadi/events/models.py

```diff
@@ -1183,6 +1184,9 @@ class Manvr(TlmEvent):
               stop_roll      Float                                  Stop roll angle after manvr
                   angle      Float                                         Maneuver angle (deg)
                one_shot      Float                            One shot attitude update (arcsec)
+          one_shot_roll      Float                                     One shot roll   (arcsec)
```

Comment:
Might be useful (to me anyway) to add the (new?) Post Maneuver Attitude Error (PMAE) description/definition in here too.